JapaneseFilter/sections/specific.txt

```diff
@@ -3034,6 +3034,7 @@ newsmomonga.com##.post div[align="center"] + br
 newsmomonga.com##.post div[align="center"] + br + hr
 nc-net.or.jp##.box-pr-image-01
 mori.nc-net.or.jp###ok_ad_side
+gyutte.jp###gt_sp_all_header
```

Comment:
It seems that there is another similar ad leftover at the bottom.
<details><summary>Screenshot</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/29142494/643d6895-5ca8-4947-965f-4a1fc8d2bbef)

</details>

Could you please hide it too?
I guess that in this case it can be done by modifying current rule.
```adblock
gyutte.jp##div[id^="gt_sp_all_"]
```